test/compute/2019-12-01/armcompute/zz_generated_connection.go

```diff
@@ -5,89 +5,4 @@
 
 package armcompute
 
-import (
-	"fmt"
-	"github.com/Azure/azure-sdk-for-go/sdk/armcore"
-	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
-)
-
-const scope = "https://management.azure.com//.default"
```

Comment:
Did something break? The content in this file was removed.

Reply:
This is expected, the only content in the file is the telemetry `const` (it probably got lost in the sea of red).